Library/Homebrew/cmd/update.rb

```diff
@@ -115,6 +163,12 @@ def load_tap_migrations
   rescue LoadError
     false
   end
+
+  def load_formula_renames
+    load 'formula_renames.rb'
```

Comment:
I'll test it again, but it can prevent FORMULA_RENAMES from updloading and we won't get new renamed formula. Hm, this is the same problem we had for `TAP_FORMULA_MIGRATIONS`. FORMULA_RENAMES can be reinitialized.